src/parser/jobs/drk/modules/Darkside.js

```diff
@@ -0,0 +1,82 @@
+import ACTIONS from 'data/ACTIONS'
+import Module from 'parser/core/Module'
+import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import React, {Fragment} from 'react'
+import {ActionLink} from 'components/ui/DbLink'
+
+const DARKSIDE_MAX_DURATION = 60000
+const DARKSIDE_EXTENSION = {
+	[ACTIONS.FLOOD_OF_SHADOW.id]: 30000,
+	[ACTIONS.EDGE_OF_SHADOW.id]: 30000,
+}
+const GCD_LENGTH = 2500
```

Comment:
Is it feasible to use the estimate GCD rather than a hard coded value?

Reply:
Definitely maybe?  All that the GCD_LENGTH constant is really being used for is "how long before we start dinging you for not having Darkside on at the start of the fight"... similar to the OGCD core module.  I renamed the constant to be a little more clear on that... if you think it's worth the effort to try to tie into the GCD estimation, I can dig further, but being off by a couple hundred milliseconds of Darkside downtime if someone isn't applying it in first weave isn't a big concern to me.

Reply:
Fair call; as long as it's only used for _first_ gcd, I don't mind that.